content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -358,6 +358,88 @@ The affinity term is applied to namespaces selected by both `namespaceSelector`
 Note that an empty `namespaceSelector` ({}) matches all namespaces, while a null or empty `namespaces` list and
 null `namespaceSelector` matches the namespace of the Pod where the rule is defined.
 
+#### matchLabelKeys
+
+{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+
+`matchLabelKeys` is an optional field which specifies the keys for the labels that should match with the incoming Pod's labels, when satisfying the Pod (anti)affinity.
+
+{{< note >}}
+The `matchLabelKeys` field is a alpha-level field and disabled by default in 1.29. 
+When you want to use it, you have to enable it via the
+`MatchLabelKeysInPodAffinity` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+{{< /note >}}
+
+The keys are used to lookup values from the pod labels, those key-value labels are ANDed 
+with labelSelector to select the group of existing pods which will be taken into Pod (anti)affinity calculation. 
+
+One common usecase would be to use `matchLabelKeys` with `pod-template-hash` in Deployment 
+so that the rolling upgrade won't break affinity.
```

Comment:
```suggestion
One likely use case would be to use `matchLabelKeys` with `pod-template-hash` (set on Pods
managed as part of a Deployment), so that a rolling upgrade won't break affinity.
```